### PR TITLE
Get token code

### PR DESCRIPTION
**Pull request by Sripathm2**

### CODE REVIEW

File: pythonAPIClient/client.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-from .camera import Camera, IPCamera, NonIPCamera, StreamCamera
-from .error import Error
+#from .camera import Camera, IPCamera, NonIPCamera, StreamCamera
+#from .error import Error
```
Comment (samyellin) [inline pythonAPIClient/client.py]: Should we just wait to merge this until those classes are complete?

File: pythonAPIClient/client.py [hunk 2]
```diff
@@ -71,3 +73,8 @@ def camera_by_id(self, cameraID):
     # replace others with desired field names
     def search_camera(self, others):
         pass
+
+if __name__ == '__main__':
+    clinetID = "dd53cbd9c5306b1baa103335c4b3e91d8b73386ba29124ea2b1d47a619c8c066877843cd8a7745ce31021a8d1548cf2a"
+    clientSecret = "f7ad9184949b914f2c73da4f6c82f6e93660c23c644e0ca4c9ac8268a141e02f258728aeefe864a6d03f709163cefe9c"
+    Client = Client(clinetID,clientSecret)
```
Comment (samyellin) [inline pythonAPIClient/client.py]: I don't think we should have this section in master. This type of code is for testing

File: pythonAPIClient/client.py [hunk 3]
```diff
@@ -9,19 +9,21 @@ class Client(object):
     # TODO: corresponding to the auth route
     @staticmethod
     def request_token(id, secret):
-        return 'dummy'
+        url = 'https://cam2-api.herokuapp.com/auth/?'
+        url += 'clientID='+id+'&clientSecret='+secret
+        response = requests.get(url)
+        return response.json()['token']
```
Comment (samyellin) [inline pythonAPIClient/client.py]: This function should probably check for the response and throw an error if there's some problem

File: pythonAPIClient/client.py (hunk 1 above)
Comment (mush-zhang) [reply pythonAPIClient/client.py]: are those import statements giving errors?

File: pythonAPIClient/client.py (hunk 3 above)
Comment (mush-zhang) [reply pythonAPIClient/client.py]: second.
you can check the status code with requests

File: pythonAPIClient/client.py (hunk 1 above)
Comment (Sripathm2) [reply pythonAPIClient/client.py]: yes they are giving errors

**Review by mush-zhang — CHANGES_REQUESTED** [review]
let's add testcases for now

**Review by mush-zhang — APPROVED** [review]

**Pull request merged**